MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: Appointment drag and drop in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about Appointment drag and drop support in Syncfusion .NET MAUI Scheduler(SfScheduler) control.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Appointment Drag and Drop in .NET MAUI Scheduler
+Appointments can be rescheduled using the drag and drop operation. To perform drag-and-drop operations within the scheduler, enable the `AllowAppointmentDrag` property of SfScheduler. You can disable appointment drag and drop by assigning `false` to `AllowAppointmentDrag` property. The default value of is `true`.
+
+{% tabs %}
+{% highlight xaml %}
+<scheduler:SfScheduler x:Name="scheduler" View="Week" AllowAppointmentDrag="false">        
+</scheduler:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+this.scheduler.AllowAppointmentDrag = false;
+{% endhighlight %}
+{% endtabs %}
+
+![Appointment Drag and Drop in .NET MAUI Scheduler](images/drag-and-drop/appointment-drag-and-drop.gif)
+
+Refer [here](https://help.syncfusion.com/maui/scheduler/appointments#creating-business-objects) to create a business object class.
+
+N> Inherit the business object class class from the `INotifyPropertyChanged` for dynamic changes in custom data.
+
+## Handle appointment drag starting
+You can get the appointment details and handle whether the appointment can be draggable or not by using `AppointmentDragStarting` event. This event will be triggered when the appointment is started dragging. The `AppointmentDragStartingEventArgs` argument contains the following properties.
+
+`Appointment` - Gets the appointment that is going to be dragged.
+`Resource` - Gets the resource under which the appointment is currently located.
```

Comment:
`Resource` - Get the resource under which the appointment is currently located.

Reply:
Content changed